.datadog/image/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.ddbuild.io/images/base/gbi-ubuntu_2204:release
+ARG TARGETARCH
+ARG TARGETOS
+
+USER root
+
+RUN apt-get update && apt-get install -y libcap2-bin
```

Comment:
This is quite large. I think the entire update/install/setcap/rm should be in the same layer

Reply:
It's complicated because the setcap relies on the binary being there. The binary is there only once it's been copied.